### DISCUSSION TITLE
How to setup a react apps with a package-based monorepo?

### DISCUSSION THREAD

**Discussion by wuarmin** [nrwl/nx#13281] [General]
Hey,
I want to create a package-based nx-monorepo (maybe using pnpm, would you recommend this decision against npm?) with multiple react SPA apps and some libs/shared-libs. I'm new to nx and want to know the best way to go? It seems that I cannot use the nx react plugin out of the box. Are plugins only useful in an integrated monorepo? Do I have to use `create-react-app` using `craco` for my setting?

Thanks for input

**Comment by nag-supratim** [comment]
would really love to know this! trying to do something similar
